src/components/FormLabel/FormLabel.module.scss

```diff
@@ -6,11 +6,28 @@
   font-size: var(--size-font-sm);
```

Comment:
@JCarmine maybe this is best served for another PR, but I'm thinking we should have this follow our pattern of component-specific variables similarly to `Button` etc.. Let me know what you think.

Reply:
Updated to use form variables, and added radio specific ones too.